pom.xml

```diff
@@ -121,6 +121,13 @@
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>ch.qos.logback</groupId>
+			<artifactId>logback-classic</artifactId>
```

Comment:
You should replace slf4j-api with this. Why only tests for production it does not work also... .